packages/grafana-ui/src/components/AutoSaveField/EllipsisAnimated.tsx

```diff
@@ -22,13 +23,34 @@ const getStyles = () => {
       display: 'inline',
     }),
     firstDot: css({
-      animation: `${firstDot} 2s linear infinite`,
+      ...handleReducedMotion(
+        {
+          animation: `${firstDot} 2s linear infinite`,
+        },
+        {
+          animation: `${firstDot} 2s linear infinite`,
```

Comment:
I think we can get rid of the animation here since it appears alongside the 'Saving' text 🤔

Reply:
maybe 🤔 

but i'm not sure we need to. everything i've read seems to state that opacity animations are fine. it's more movement/rotation animations that are potentially triggering. that's also what i've updated the rule documentation to state.

Reply:
Yes, they are usually related to movement/rotation animations rather than opacity 👌 
I think this answers also my following comment 😄 It's ok to me then, since we can handle them individually whenever we get specific feedback about one of them